src/uct/ib/base/ib_device.h

```diff
@@ -385,6 +388,10 @@ uct_ib_device_create_ah_cached(uct_ib_device_t *dev,
                                struct ibv_ah_attr *ah_attr, struct ibv_pd *pd,
                                const char *usage, struct ibv_ah **ah_p);
 
+ucs_status_t uct_ib_device_get_ndev_name(const char *dev_name, uint8_t port_num,
```

Comment:
Why do you need extra function here. I guess you can use `uct_ib_device_get_roce_ndev_name`.